src/embezzler.ts

```diff
@@ -7,11 +7,17 @@ import {
   haveEquipped,
   inebrietyLimit,
   itemAmount,
+  meatDropModifier,
```

Comment:
Style question: To what extent is meatDropModifier different from numericModifier("Meat Drop")/getModifier("Meat Drop")? Is there a compelling reason for us to consistently pick one of these?

Reply:
No idea the difference, but we should definitely pick one to consistently use.